cli/tests/unit/README.md

```diff
@@ -26,14 +26,14 @@ unitTest(
 
 ## Running tests
 
-There are three ways to run `unit_test_runner.ts`:
+There are two ways to run `unit_test_runner.ts`:
 
 ```sh
 # Run all tests.
-target/debug/deno test --allow-all --unstable cli/tests/unit/
+(cd cli/tests/testdata && ../../target/debug/deno test --allow-all --unstable ../unit)
```

Comment:
Actually, I think it is unnecessary for the unit tests to change their CWD. I will revert this.

Reply:
It's preferable that unit tests work independently of CWD

Reply:
That would be nice. Maybe in a future change. This is reverted now and the PR is good to go.